pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -447,6 +455,14 @@ type BodyAlertingConfig struct {
 	Body PostableUserConfig
 }
 
+// swagger:parameters RouteGetGrafanaSuccessfullyAppliedAlertingConfigs
+type RouteGetGrafanaSuccessfullyAppliedAlertingConfigsParams struct {
+	// Limit response to X number of alerting configurations.
+	// in:query
+	// required: true
```

Comment:
Required query parameters are semantically very awkward, users of the API never expect this. We should instead define a default limit, and use that when this is not supplied.